ssh/entrypoint.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$SSH_ENABLE" == "1" ]]; then
     chmod 600 -R /home/postgres/.ssh/id_rsa
 
     echo '>>> STARTING SSH SERVER...'
-    /usr/sbin/sshd -D &
+    /usr/sbin/sshd 2>&1
```

Comment:
sshd is daemonized by default. `-D` specify that sshd will not detach and then you force it by `&`. Maybe I've just overlooked smth